eZ/Publish/Core/REST/Common/FieldTypeProcessor/RelationListProcessor.php

```diff
@@ -12,10 +12,51 @@
 
 use eZ\Publish\Core\REST\Common\FieldTypeProcessor;
 use eZ\Publish\Core\FieldType\RelationList\Type;
+use Symfony\Component\Routing\RouterInterface;
 
 class RelationListProcessor extends FieldTypeProcessor
 {
     /**
+     * @var \Symfony\Component\Routing\RouterInterface
+     */
+    private $router;
+
+    public function setRouter(RouterInterface $router)
+    {
+        $this->router = $router;
+    }
+
+    /**
+     * In addition to the list of destinationContentIds, adds a destinationContentHrefs
+     * array, with matching content uris.
+     *
+     * @param array $outgoingValueHash
+     *
+     * @return array
+     */
+    public function postProcessValueHash($outgoingValueHash)
+    {
+        if (!isset($outgoingValueHash['destinationContentIds']) || !is_array($outgoingValueHash['destinationContentIds'])) {
+            return $outgoingValueHash;
+        }
+
+
+        if (isset($this->router)) {
```

Comment:
nitpick, but can swap the logic with `empty($this->router)` / `!isset($this->router)` to a return early pattern so code below is one level less intendent